dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -542,6 +542,8 @@ def perform_environment_checks(quiet: bool = False):
     check_docker_version(quiet)
     check_docker_compose_version(quiet)
     check_executable_entrypoint_permissions(quiet)
+    if not quiet:
```

Comment:
Non-blocking nit: Do we need to align the check with line `534` ?
```suggestion
    if get_verbose() and not quiet:
```

Reply:
No - not really - I think the Python version information is important and useful enough to be printed always - not only when the `--verbose` flag is used. 

I added the `get_verbose()` for executable check because essentially it adds exactly 0 value when printed. If it works, it's always the same message without any extra bit of information, so printing it always makes no sense.

The `--verbose` thing makes sense when you are debugging or seeing weird behaviour - because you can see if weird things happens before or after the message, so printing it is useful, but printing it in regular case makes littie sense.

On the other hand printing the detailed Python version on host is extremely useful - we've seen cases in the past where people were using Python 3.14 (not allowed now) for breeze and it causes some weird errors - and the only way we could see it's pythong 3.14 was that we saw sources for python coming from `/var/lib/python314` in stacktrace. Printing it explicitly always might help if a user has problem and copy&paste the regular output for help.